llama_stack/ui/components/chat-completions/chat-completion-detail.test.tsx

```diff
@@ -0,0 +1,193 @@
+import React from "react";
```

Comment:
nice, thanks for the tests. I am not a React tests expert for sure, but it is good to have stuff from the get go!